app/javascript/article-form/components/ImageUploader.jsx

```diff
@@ -316,16 +318,31 @@ export const ImageUploader = ({
     }
   }
 
+  function formatUploadedImages(links) {
+    let articleImageLinks = "";
+
+        links.forEach(link => {
+          const formattedLink = `![Image description](${link}) \n`;
+
+          articleImageLinks = articleImageLinks + formattedLink;
+        });
+    
+        onImageUploadSuccess?.(articleImageLinks);
+  }
+
   function handleInsertImageUploadSuccess(response) {
     dispatch({
       type: 'upload_image_success',
       payload: { insertionImageUrls: response.links },
     });
 
-    onImageUploadSuccess?.(`![Image description](${response.links})`);
```

Comment:
We've had a lot of problems with flaky tests lately, so apologies for not catching this sooner, but the tests aren't happy with this change. It looks like a relatively minor syntax change, it ought to be possible to either update the PR to restore the expected syntax or update the tests to match the new format.